sdcm/sct_runner.py

```diff
@@ -115,6 +117,83 @@ def install_prereqs(self, public_ip):
         else:
             raise Exception("Unable to install required packages:\n%s" % (result.stdout + result.stderr))
 
+    def _image(self, image_type=ImageType.source):
+        return NotImplementedError()
+
+    @property
+    def source_image(self):
+        return self._image(image_type=ImageType.source)
+
+    @property
+    def image(self):
+        return self._image(image_type=ImageType.general)
+
+    def _create_instance(self, instance_type, base_image, tags_list, instance_name=None, region_az="", test_duration=None):
+        return NotImplementedError()
+
+    def tag_image(self, image_id, image_type):
+        return NotImplementedError()
+
+    def create_image(self):
+        return NotImplementedError()
+
+    def _get_base_image(self, image=None):
+        return NotImplementedError()
```

Comment:
abstract

Reply:
Currently pylint pick ip methods with only return NotImplementedError() and treat them as abstract, so we good here